samcli/commands/_utils/custom_options/hook_package_id_option.py

```diff
@@ -82,7 +82,7 @@ def handle_parse_result(self, ctx, opts, args):
                 LOG.info("Running Prepare Hook to prepare the current application")
 
                 iac_project_path = os.getcwd()
-                output_dir_path = os.path.join(iac_project_path, ".aws-sam", "iacs_metadata")
+                output_dir_path = os.path.join(iac_project_path, ".aws-sam-iacs", "iacs_metadata")
```

Comment:
Yes. This was something we discussed to as a workaround for `sam build` since by default the `.aws-sam` directory won't be copied over to the scratch directory in lambda builders. This bypasses that so that the Makefile will be copied to where we do the building.